src/Date/Local.elm

```diff
@@ -0,0 +1,111 @@
+module Date.Local exposing (..)
+
+{-| Record to store localized time format.
+
+Time zones and default formats are not implemented,
+but included to avoid possible version conflicts in the future.
+-}
+
+type alias Local =
+  { date :
+    { months : Months
+    , monthsAbbrev : Months
+    , wdays : WDays
+    , wdaysAbbrev : WDays
+    , defaultFormat : Maybe String
+    }
+  , time :
+    { am : String
+    , pm : String
+    , defaultFormat : Maybe String
+    }
+  , timeZones : Maybe TimeZones
+  , defaultFormat : Maybe String
+  }
+
+type alias Months =
+  { jan : String
+  , feb : String
+  , mar : String
+  , apr : String
+  , may : String
+  , jun : String
+  , jul : String
+  , aug : String
+  , sep : String
+  , oct : String
+  , nov : String
+  , dec : String
+  }
+
+type alias WDays =
+  { mon : String
+  , tue : String
+  , wed : String
+  , thu : String
+  , fri : String
+  , sat : String
+  , sun : String
+  }
+
+type alias TimeZones = Never
```

Comment:
I appreciate the intention of letting this be defined later, but it's exposed and once released cannot be changed without a major bump. Even though `Maybe Never` only has one value, `Nothing`. The versioner isn't that smart.